corpora/proverbs.dep

```diff
@@ -27,3 +28,979 @@
+"<,>"
+	"," cm @punct #5->11
+"<кийинки>"
+	"кийин" adv attr @amod #6->7
+"<көч да>"
+	"көч" n nom @nsubj #7->11
+		"да" postadv @advmod #8->11
+"<ошол>"
+	"ошол" det dem @det #9->10
+"<жакка>"
+	"жак" n dat @obl #10->11
+"<барат>"
+	"бар" v iv aor p3 sg @root #11->0
+"<.>"
+	"." sent @punct #12->11
+
+# sent_id = proverbs:25
+# author:id = Сыргабековна&Сабыровна:499
+# text = Эшекке төшөк жарашпайт.
+# text[eng] = A sleeping mat is not suitable for a donkey.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Эшекке>"
+	"эшек" n dat @obl #1->3
+"<төшөк>"
+	"төшөк" n nom @nsubj #2->3
+"<жарашпайт>"
+	"жара" v tv coop neg aor p3 sg @root #3->0
+"<.>"
+	"." sent @punct #4->3
+
+# sent_id = proverbs:26
+# author:id = Сыргабековна&Сабыровна:275
+# text = Кетмен чаппай - нан жок, чеке терсиз - мал жок.
+# text[eng] = Without the hoe striking there's no bread, without a sweaty forehead there's no livestock.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Кетмен>"
+	"кетмен" n nom @csubj #1->2
+"<чаппай>"
+	"чап" v tv neg gna_perf @advcl #2->5
+"<->"
+	"-" guio @punct #3->5
+"<нан>"
+	"нан" n nom @nsubj #4->5
+"<жок>"
+	"жок" adj @parataxis #5->13
+		"э" cop aor p3 sg @cop #6->5
+"<,>"
+	"," cm @punct #7->13
+"<чеке>"
+	"чеке" n nom @csubj #8->9
+"<терсиз>"
+	"тер" n @advcl #9->13
+		"сыз" post @case #10->9
+"<->"
+	"-" guio @punct #11->13
+"<мал>"
+	"мал" n nom @nsubj #12->13
+"<жок>"
+	"жок" adj @root #13->0
+		"э" cop aor p3 sg @cop #14->13
+"<.>"
+	"." sent @punct #15->13
+
+# sent_id = proverbs:27
+# author:id = Сыргабековна&Сабыровна:246
+# text = Иштесең көл болот, иштебесең чөл болот.
+# text[eng] = If you work there will be a lake, if you don't work there will be a desert.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Иштесең>"
+	"иште" v tv gna_cond p2 sg @advcl #1->3
+"<көл>"
+	"көл" n nom @nsubj #2->3
+"<болот>"
+	"бол" v iv aor p3 sg @conj #3->7
+"<,>"
+	"," cm @punct #4->7
+"<иштебесең>"
+	"иште" v tv neg gna_cond p2 sg @advcl #5->7
+"<чөл>"
+	"чөл" n nom @nsubj #6->7
+"<болот>"
+	"бол" v iv aor p3 sg @root #7->0
+"<.>"
+	"." sent @punct #8->7
+
+# sent_id = proverbs:28
+# author:id = Сыргабековна&Сабыровна:5
+# text = Адамды кийиминен тааныбайт, ишинен тааныйт.
+# text[eng] = (One) doesn't know a person by his clothes, (one) knows (him) by his work.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Адамды>"
+	"Адам" n nom @nsubj #1->3
+"<кийиминен>"
+	"кийим" n px3sp abl @obl #2->3
+"<тааныбайт>"
+	"тааны" v tv neg aor p3 sg @parataxis #3->6
+"<,>"
+	"," cm @punct #4->6
+"<ишинен>"
+	"иш" n px3sp abl @obl #5->6
+"<тааныйт>"
+	"тааны" v tv aor p3 sg @root #6->0
+"<.>"
+	"." sent @punct #7->6
+
+# sent_id = proverbs:29
+# author:id = Сыргабековна&Сабыровна:114
+# text = Бир ит көрүп үрөт, бир ит көрбөй үрөт.
+# text[eng] = One dog sees (something) and barks, one dog doesn't see (anything) and barks.
+# label_cg = JNW_checked
+# label_dep = to_check
+"<Бир>"
+	"бир" num @nummod #1->2
+"<ит>"
+	"ит" n nom @csubj #2->3
```

Comment:
`@nsubj #2->4`.  In terms of the dependency on `4` and not `3`, it's because it's more along the lines of the English parse "one dog barks seeing, one dog barks not seeing".  The English translation with a conjunction isn't really how it's working structurally in Kyrgyz; it just offers a more fluent gloss.